jzp-code/kukdo/project/cloud-storage/cloud-storage-parent/cloud-storage-web/src/main/java/com/xiaomi/service/impl/LocalStorageServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.xiaomi.service.impl;
+
+import com.xiaomi.service.StorageService;
+
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+@Service("LocalStorage")
+public class LocalStorageServiceImpl implements StorageService {
+
+    private final Path rootLocation = Paths.get("upload_file");

```

Comment:
should be configurable

Reply:
> should be configurable

yes, here just for test yesterday, I will add a property in `application.properties` and update this part.